org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,440 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: webappreader.sh
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home  # to be able to save html pages to a hard drive
+  - --share=network # it is necessary to download html from the Internet
+  - --device=dri # For the Qt webengine module to work perfectly with gpu
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess # necessary for QtWebEngineProcess to work
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: wget2 #  proper workable version = 2.1
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap
+      - ./configure --prefix=/app
+      - make
+      - make install
+    sources:
+      - type: git
+        url: https://gitlab.com/gnuwget/wget2.git
+        commit: 657e3d22c488a2d1f173e208e253995e15f8b6db
+        
+  - name: webappreader
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/webappreader/
+      - cp -r database /app/webappreader/database
+      - cp -r pages /app/webappreader/pages
+      - cp -r src /app/webappreader/src
+      - cp -r ui /app/webappreader/ui
+      - install -Dm644 constants.py -t /app/webappreader
+      - install -Dm644 settings.py -t /app/webappreader
+      - install -Dm644 tools.py -t /app/webappreader
+      - install -Dm644 webappreader.py -t /app/webappreader
+    sources:
+      - type: git
+        url: https://github.com/ShvonderSiN/WebAppReader.git
+        commit: 8577a848c3032e298c5f44444221a294ca19f25c
+      - type: script
+        commands:
+          - 'python3 /app/webappreader/webappreader.py "$@"'
+        dest-filename: webappreader.sh
+    post-install:
+      - install -Dm644 src/logo_256.png /app/share/icons/hicolor/256x256/apps/org.mintguide.WebAppReader.png
+      - install -Dm755 webappreader.sh /app/bin/webappreader.sh
+      - install -Dm644 flatpak/org.mintguide.WebAppReader.desktop /app/share/applications/org.mintguide.WebAppReader.desktop
+      - install -Dm644 flatpak/org.mintguide.WebAppReader.metainfo.xml /app/share/metainfo/org.mintguide.WebAppReader.metainfo.xml
+      
+  # depencies
+  - {"name": "python3-requirements",
```

Comment:
JSON in YAML?

Reply:
This entire part should be in a separate json file, as it is autogenerated by flatpak-pip-generator.

Reply:
I just wanted the build to be, just one manifest file, so I put everything in the manifest.. If that's not allowed, I'll do as you say...Should I put json dependencies in this pull request next to the manifest?

Reply:
so I made the json file an additional module and put it next to the manifest